File: Maths_And_Stats/Geometry/Manhattan_Distance/manhattan_dist.py
def calcManhattanDist(x1, y1, x2, y2) -> float:
    return abs(x1 - x2) + abs(y1 - y2)

def main():
    print(calcManhattanDist(2, 5, 10, 14))

if __name__ == "__main__":
    main()

File: Maths_And_Stats/Geometry/Manhattan_Distance/manhattan_dist.go
package main

import (
	"fmt"
	"math"
)

func calcManhattanDist(x1, y1, x2, y2 int) {
	dist := math.Abs(float64(x1 - x2)) + math.Abs(float64(y1 - y2))
	fmt.Println(dist)
}

func main() {
	calcManhattanDist(2, 5, 10, 14)
}